lib/membrane_rtmp_plugin/rtmp_server/client_handler.ex

```diff
@@ -96,6 +109,31 @@ defmodule Membrane.RTMP.Server.ClientHandler do
     {message_handler_state, events} =
       MessageHandler.handle_client_messages(messages, state.message_handler_state)
 
+    state =
+      if message_handler_state.publish_msg != nil and not state.notified_about_client? do
+        %{publish_msg: %Membrane.RTMP.Messages.Publish{stream_key: stream_key}} =
+          message_handler_state
+
+        if is_function(state.new_client_callback) do
+          state.new_client_callback.(self(), state.app, stream_key)
+        else
+          raise "new_client_callback is not a function"
+        end
+
+        with {:error, reason} <-
+               :timer.send_after(
```

Comment:
[NIT] You can use a function form Elixir module as well:
```suggestion
               Process.send_after(
```

Reply:
BTW. does it happen that this function fails? I assume, that it's not blocking